src/components/PassengerShowcaseForm.js

```diff
@@ -266,8 +269,13 @@ const PassengerShowcaseForm = () => {
         {error && <div className={css.error}>{error}</div>}
         {submitted && (
           <div className={css.submitted}>
-            Thank you for submitting to the Passenger Showcase! Please refresh
-            the page in order to upload another submission.
+            Thank you for submitting to the Passenger Showcase!{' '}
+            <a
+              href={`https://github.com/CodingTrain/thecodingtrain.com/pull/${githubPRNo}`}>
+              Click here
+            </a>{' '}
+            to view your submission. Please refresh the page in order to upload
+            another submission.
```

Comment:
```suggestion
            Thank you for submitting to the Passenger Showcase!{' '}
            <a
              href={`https://github.com/CodingTrain/thecodingtrain.com/pull/${githubPRNo}`}>
              You can follow this linked pull request on GitHub for more.
            </a>{' '}
            Please refresh the page in order to upload another submission.
```

Reply:
It's best to avoid "click here" language for accessibility! 

https://webaccess.berkeley.edu/ask-pecan/click-here